api/src/main/java/openconsensus/OpenConsensus.java

```diff
@@ -0,0 +1,49 @@
+package openconsensus;
+
+import java.util.concurrent.atomic.AtomicReference;
+import openconsensus.trace.Tracer;
+
+/**
+ * Registry provides a global access to observability objects.
+ * An implementation is responsible for setting the Registry.
+ *
+ * @author Pavol Loffay
+ */
+public class OpenConsensus {
```

Comment:
do we need an ability to construct `OpenConsensus` to use in tests? Or we will be OK with individual components constructions onle?

Reply:
Do you mean our tests or instrumentation? There is a reset method on the `OpenConsensus` to revert it to the default.